src/cli/seat_test.rs

```diff
@@ -583,6 +584,63 @@ async fn run(seat_test: Rc<SeatTest>) {
         }
         println!();
     });
+    let st = seat_test.clone();
+    TouchDown::handle(tc, se, (), move |_, ev| {
+        if all || ev.seat == seat {
```

Comment:
The events should be collected and only be printed on the next frame event. Similar to the tablet frame event.